webapi/src/main/scala/org/knora/webapi/responders/v1/StandoffResponderV1.scala

```diff
@@ -76,6 +77,10 @@ class StandoffResponderV1 extends ResponderV1 {
       */
     private def createMappingV1(xml: String, userProfile: UserProfileV1): Future[CreateMappingResponseV1] = {
 
+        // http://stackoverflow.com/questions/7087353/can-a-scala-for-loop-modify-variables-outside-its-scope
+        // FIXME Ben, I am so ugly and non functional style
```

Comment:
@benjamingeer Could you give me a hand here?

I would like to delete the temp file that I created in the for comprehension.